website/docs/d/pi_catalog_images.html.markdown

```diff
@@ -19,13 +19,12 @@ data "ibm_pi_catalog_images" "ds_images" {
 ```
 
 **Notes**
-* Please find [supported Regions](https://cloud.ibm.com/apidocs/power-cloud#endpoint) for endpoints.
-* If a Power cloud instance is provisioned at `lon04`, The provider level attributes should be as follows:
-  * `region` - `lon`
-  * `zone` - `lon04`
-  
-  Example usage:
+- Please find [supported Regions](https://cloud.ibm.com/apidocs/power-cloud#endpoint) for endpoints.
+- If a Power cloud instance is provisioned at `lon04`, The provider level attributes should be as follows:
+  - `region` - `lon`
+  - `zone` - `lon04`
   
+Example usage:
```

Comment:
The change in alignment is not needed. This is supposed to be part of the second bullet point.

Reply:
It is needed if you look at it in the doc preview tool:
https://registry.terraform.io/tools/doc-preview#example-usage

WIthout Change:
![Screenshot 2024-01-12 at 08-43-28 ibm_pi_catalog_images Data Sources IBM-Cloud_ibm Terraform Terraform Registry](https://github.com/IBM-Cloud/terraform-provider-ibm/assets/90468712/ad573131-e8df-49a1-917b-0702789d9175)

With Change:
![Screenshot 2024-01-12 at 08-44-52 Doc Preview Tool Terraform Registry](https://github.com/IBM-Cloud/terraform-provider-ibm/assets/90468712/63b5c20f-b540-406a-98a0-3a8ca2b1f50c)

Reply:
There lots of inconsistencies and errors in the existing markdown files.